src/Environments/Windows/Win32M68kPlatform.cs

```diff
@@ -22,12 +22,13 @@ public override string DefaultCallingConvention
 			get { return ""; }
 		}
 
-		public override HashSet<RegisterStorage> CreateImplicitArgumentRegisters()
-		{
-			return new HashSet<RegisterStorage>
-			{
-			};
-		}
+        public override HashSet<RegisterStorage> CreateImplicitArgumentRegisters()
+        {
+            return new HashSet<RegisterStorage>
+            {
+                Architecture.StackRegister,
+            };
+        }
 
```

Comment:
Same comment here; `Platform`'s `CreateImplicitArgumentRegisters` already does this.